common/Timer.cc

```diff
@@ -36,9 +36,13 @@ FlowId Timer::getFlowEdge() {
 
 Timer::~Timer() {
     auto clock = chrono::steady_clock::now();
-    auto dur = std::chrono::duration<double, std::milli>(clock - start);
-    log.debug("{}: {}ms", this->name.str, dur.count());
-    sorbet::timingAdd(this->name, start, clock, move(args), self, prev);
+    auto dur = clock - start;
+    if (dur > std::chrono::microseconds(250)) {
+        // the trick ^^^ is to skip double comparison in the common case and use the most efficient represnetation.
```

Comment:
You probably use Timers more than anyone else so i'm fine with you wanting this change.

My question is "Could this be a footgun for someone who doesn't really understand what our Timers do?"

Reply:
It's a footgun in case someone tries to measure something very short with Timers.
But this a footgun already because Timers themselves have overhead and if you shouldn't be measuring something short with them.